rdr_service/resource/generators/participant.py

```diff
@@ -707,9 +824,12 @@ def _prep_physical_measurements(self, p_id, ro_session):
                 'finalized_site': self._lookup_site_name(row.finalizedSiteId, ro_session),
                 'finalized_site_id': row.finalizedSiteId,
             })
+            _act(row.finalized, ActivityGroup.Profile, ParticipantActivity.PhysicalMeasurements)
```

Comment:
Should the result of the `_act` call be appended to the `activity` list?

Reply:
Yes, good catch.